vendor/k8s.io/kubernetes/cmd/kube-apiserver/app/options/options_test.go

```diff
@@ -109,6 +109,8 @@ func TestAddFlags(t *testing.T) {
 			MaxMutatingRequestsInFlight: 200,
 			RequestTimeout:              time.Duration(2) * time.Minute,
 			MinRequestTimeout:           1800,
+			JSONPatchMaxCopyBytes:       int64(3 * 1024 * 1024),
```

Comment:
this is not the upstream commit, but adapted to the different order. This makes it hard to follow and compare. Better pick in the same order as upstream.